pontoon/base/models.py

```diff
@@ -29,7 +29,10 @@ class UserProfile(models.Model):
 
 def validate_cldr(value):
     for item in value.split(','):
-        number = int(item.strip())
+        try:
+            number = int(item.strip())
+        except ValueError:
+            return
```

Comment:
Since this overrides the integer check you should raise a ValidationError here complaining that it's not an integer.